$NetBSD: patch-lib_Makefile.local,v 1.3 2015/08/26 13:33:36 wiz Exp $

* On Mac OS X (Darwin), for the install_name of the notmuch shared
  library, use the full path rather than just the file name.  This
  enables programs that link against it as a dependent library to find
  it when they run.

  Upstream bug report:

    http://notmuchmail.org/pipermail/notmuch/2014/018956.html

* Don't overlap with BSD's libutil.

--- lib/Makefile.local.orig	2015-06-27 13:05:31.000000000 +0000
+++ lib/Makefile.local
@@ -76,8 +76,8 @@ libnotmuch_modules := $(libnotmuch_c_src
 $(dir)/libnotmuch.a: $(libnotmuch_modules)
 	$(call quiet,AR) rcs $@ $^
 
-$(dir)/$(LIBNAME): $(libnotmuch_modules) notmuch.sym util/libutil.a parse-time-string/libparse-time-string.a
-	$(call quiet,CXX $(CXXFLAGS)) $(libnotmuch_modules) $(FINAL_LIBNOTMUCH_LDFLAGS) $(LIBRARY_LINK_FLAG) -o $@ util/libutil.a parse-time-string/libparse-time-string.a
+$(dir)/$(LIBNAME): $(libnotmuch_modules) notmuch.sym util/libmyutil.a parse-time-string/libparse-time-string.a
+	$(call quiet,CXX $(CXXFLAGS)) $(libnotmuch_modules) $(FINAL_LIBNOTMUCH_LDFLAGS) $(LIBRARY_LINK_FLAG) -o $@ util/libmyutil.a parse-time-string/libparse-time-string.a
 
 notmuch.sym: $(srcdir)/$(dir)/notmuch.h $(libnotmuch_modules)
 	sh $(srcdir)/$(lib)/gen-version-script.sh $< $(libnotmuch_modules) > $@
